### PR TITLE
More helpful error message on missing public token

### PR DESCRIPTION
**Pull request by maackle**

### CODE REVIEW

**Comment by zippy** [comment]
Interesting.  This last addition breaks a few tests.  Apparently we have tests on DNAs with no zomes!

**Comment by zippy** [comment]
Looks like there's still an "Attempting to initialize DNA with zero zomes!"

File: core_types/src/error/error.rs [hunk 1]
```diff
@@ -0,0 +1,369 @@
+use self::HolochainError::*;
```
Comment (maackle) [inline core_types/src/error/error.rs]: Uh oh, did this file renamed and then re-added?

Comment (maackle) [reply core_types/src/error/error.rs]: Removed it again in https://github.com/holochain/holochain-rust/pull/1240

**Pull request merged**